composer.json

```diff
@@ -35,6 +35,7 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-dom": "*",
         "ext-json": "*",
+        "psr/container": "^2.0.2",
         "psr/http-factory": "^1.0",
```

Comment:
Relax container range to allow 1.x as well. Differences are minimal but most libraries wanting container interface require 1.x still

Reply:
IMO no: `1.x` is not needed, as we already dropped support for PHP:<7.4, so consumers can safely upgrade to `psr/container:^2`

Reply:
They can but this will unnecessary limit installation of unrelated libraries still wanting container `^1.0` over  which consumers have no or limited control.

Reply:
> Since we don't even support v2 in laminas-servicemanager yet (pending updates to container-interop)

Urgh, I was unaware of this, sorry :| 

That's indeed a problem.

Reply:
6d41494a3e70cf3e2f5c832fead20e4036c49320